examples/recsys2020.ipynb

```diff
@@ -300,76 +462,131 @@
-    "    # Fill in na/missing values in target columns + hashtags, domains, links\n",
-    "    nvt.ops.FillMissing(columns=label_name + ['hashtags', 'domains', 'links']),\n",
-    "    # Change data type to uint32 to save memory/storage\n",
-    "    nvt.ops.LambdaOp(\n",
-    "        op_name='astypeint32',\n",
-    "        f=lambda col, gdf: col.astype(np.uint32),\n",
-    "        columns=label_name + ['timestamp',\n",
-    "                              'a_follower_count', \n",
-    "                              'a_following_count',\n",
-    "                              'a_account_creation',\n",
-    "                              'b_follower_count',\n",
-    "                              'b_following_count',\n",
-    "                              'b_account_creation'],\n",
-    "        replace=True\n",
-    "    ),\n",
-    "    # Apply splitmedia function\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We apply splitmedia function to split the media."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "proc.add_feature(\n",
     "    nvt.ops.LambdaOp(\n",
     "        op_name='splitmedia',\n",
-    "        f=lambda col, gdf: splitmedia2(col),\n",
+    "        f=lambda col, gdf: splitmedia(col),\n",
     "        columns=['media'],\n",
     "        replace=True\n",
-    "    ),\n",
-    "    # Encode columns in continous integer to save memory\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We fill in na/missing values in target columns + hashtags, domains, links."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "proc.add_feature(\n",
+    "    nvt.ops.FillMissing(columns=label_name + ['hashtags', 'domains', 'links'])\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We encode columns in continous integer to save memory. Some categorical columns contain long hashes of type String as values to preserve the privacy of the users (e.g. userId, language, etc.). Long hashes of type String requires signficant amount of memory to store. We encode/map the Strings to continous Integer to save significant memory. Finally, we optimize the datatype."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "proc.add_preprocess([\n",
     "    nvt.ops.Categorify(\n",
     "        columns=['media', 'language', 'tweet_type', 'tweet_id', \n",
     "                 'a_user_id', 'b_user_id', 'hashtags', 'domains', 'links']\n",
     "    ),\n",
-    "    # Change data type to uint32 to save memory/storage\n",
     "    nvt.ops.LambdaOp(\n",
-    "        op_name='astypeint32_2',\n",
+    "        op_name='astypeint32',\n",
```

Comment:
Do we even need this lambda op? Alberto has added support for setting the dtypes we write out columns as - can we use that instead? https://github.com/NVIDIA/NVTabular/pull/392

Reply:
Yes, that is a good point. I updated the notebook using #392